clinica/pipelines/cli_param/option.py

```diff
@@ -185,3 +185,10 @@
     type=click.Path(exists=True, dir_okay=True, resolve_path=True),
     help="Compute atlases at the end of the path",
 )
+
+delete_cache = option(
+    "-dc",
+    "--delete_cache",
+    is_flag=True,
+    help="Use a lighter version of the pipeline",
```

Comment:
Could you add a bit more information in the description?
Maybe something like: "If True, large intermediary files will be deleted before the end of the pipeline. This option is only implemented in dwi_preprocessing_using_t1 for now."

Reply:
I like this description 👍

Reply:
> "This option is only implemented in dwi_preprocessing_using_t1 for now."

With this as a comment, rather than in the description. Users don't care about this.

Reply:
Sure !